limitador/src/lib.rs

```diff
@@ -375,27 +375,29 @@ impl RateLimiter {
         namespace: &Namespace,
         values: &HashMap<String, String>,
         delta: i64,
-    ) -> Result<bool, LimitadorError> {
+    ) -> Result<(bool, Vec<Counter>, Option<String>), LimitadorError> {
```

Comment:
I think it'd be better to add a new method for that path... At the library level (i.e. what's in `limitador/limitador` being able to chose whether to pay the overhead of being able to get that data should be a "first class citizen"

Reply:
Do you guys have perf tests to measure what the additional overhead would be?

Reply:
Yes, have a redis running, and run `cargo bench`